vpp-manager/startup/startup.go

```diff
@@ -115,13 +115,21 @@ func parseEnvVariables(params *config.VppManagerParams) (err error) {
 		hooks.RegisterBashHook(hooks.BEFORE_IF_READ, conf)
 	}
 
+	// NetworkManager wipes out DNS config when the uplink interface
+	// is moved. We counter it by telling it to leave the DNS config alone.
+	hooks.RegisterBashHook(hooks.BEFORE_VPP_RUN, hooks.FIX_DNS_SCRIPT)
```

Comment:
Maybe that'd be easier with renaming `network_restart.sh` : `default_hook.sh` and passing the hookpoint when calling it.
That way we can start the script with
````bash
if [ $1 = "BEFORE_VPP_RUN" ]; then 
   ...
elif ....
````
and have everything in a single glance.

Reply:
Great idea. Thanks.